src/dos/program_loadfix.cpp

```diff
@@ -90,22 +90,26 @@ void LOADFIX::AddMessages() {
 	        "Load a program in the specific memory region and then run it.\n"
 	        "\n"
 	        "Usage:\n"
-	        "  [color=light-green]loadfix[reset] [color=light-cyan]GAME[reset] [color=white][PARAMETERS][reset]\n"
+	        "  [color=light-green]loadfix[reset] [color=white][-SIZE][reset] [color=light-cyan]GAME[reset] [color=white][PARAMETERS][reset]\n"
 	        "  [color=light-green]loadfix[reset] [/d] (or [/f])[reset]\n"
 	        "\n"
 	        "Parameters:\n"
-	        "  [color=light-cyan]GAME[reset]  game or program to load, optionally with parameters\n"
+	        "  [color=light-cyan]GAME[reset]\t\tgame or program to load, optionally with parameters\n"
+	        "  [color=white]-SIZE[reset]\t\tSIZE indicates the number of kilobytes to be allocated\n"
+	        "  /d (or /f)\tFrees the previously allocated memory."
 	        "\n"
 	        "Notes:\n"
-	        "  - The most common use cases of this command are to fix DOS games or programs\n"
-	        "    which show either the \"[color=white]Packed File Corrupt[reset]\" or [color=white]\"Not enough memory\"[reset] (e.g.,\n"
-	        "    from some 1980's games such as California Games II) error message when run.\n"
+	        "  - The most common use case of this command is to fix DOS games such as\n"
```

Comment:
minor: Of course DOS games, as this is DOSBox. So I'd just say "games" 😄

Reply:
It also runs WordPerfect 5.1 (or WordStar, for that matter), doesn't it? :wink: I would not be surprised if DOSBox is also used to run legacy DOS-programs under 64-bit Windows.